onboarding.md

```diff
@@ -20,7 +20,7 @@
 
 #  1. <a name='Prerequisites'></a>Prerequisites
 
-If you have followed the [general onboarding guide](https://github.com/mojaloop/mojaloop/blob/master/onboarding.md#mojaloop-onboarding), you should already have the following cli tools installed:
+If you have followed the [general onboarding guide](https://github.com/mojaloop/mojaloop/blob/main/onboarding.md#mojaloop-onboarding), you should already have the following cli tools installed:
```

Comment:
I migrated master --> main on the https://github.com/mojaloop/mojaloop repo to make sure this works